src/lib/ERC7498NFTRedeemables.sol

```diff
@@ -393,13 +411,17 @@ contract ERC7498NFTRedeemables is IERC7498, DynamicTraits, RedeemablesErrors {
                 // Get the substandard value and place on the stack.
                 bytes32 substandardValue = traitRedemptions[i].substandardValue;
 
+                // Get the token and place on the stack.
+                address token = traitRedemptions[i].token;
+
                 // Get the trait key and place on the stack.
                 bytes32 traitKey = traitRedemptions[i].traitKey;
 
+                // Get the trait value and place on the stack.
                 bytes32 traitValue = traitRedemptions[i].traitValue;
 
                 // Get the current trait value and place on the stack.
-                bytes32 currentTraitValue = getTraitValue(traitKey, identifier);
+                bytes32 currentTraitValue = DynamicTraits(token).getTraitValue(identifier, traitKey);
```

Comment:
let's use IERC7496 (can import from shipyard) everywhere instead of DynamicTraits